concourse/pipelines/guest-package-build.yaml

```diff
@@ -1596,6 +1596,489 @@ jobs:
       result_state: "failure"
       start_timestamp: ((.:start-timestamp-ms))
 
+- name: build-compute-image-windows
```

Comment:
non blocking but i would think this much change merits adding templates; it was the difficulty of reviewing long repeated sections that prompted us to do so.

Reply:
Fixed

Reply:
Ack, we talked about this topic in our 1:1